base_layer/core/tests/tests/header_sync.rs

```diff
@@ -267,7 +267,7 @@ async fn test_header_sync_uneven_headers_and_blocks_peer_lies_about_pow_no_ban()
     match event {
         StateEvent::HeadersSynchronized(_val, sync_result) => {
             assert_eq!(sync_result.headers_returned, 0);
-            assert_eq!(sync_result.peer_fork_hash_index, 3);
+            assert_eq!(sync_result.peer_fork_hash_index, 5);
```

Comment:
whats going here, why did this value change? To me 3 sounds like the correct number here?